### PR TITLE
Render markdown in button labels

### PR DESCRIPTION
**Pull request by stoeffel**
We want to support markdown in buttons for Outline MC/Draggable questions. 

### Context
Currently they use `Nri.Button.submit` which takes html, to render markdown. We're moving away from submitting the answer as a form submit, that's way we need to switch to `Nri.Button.button`.

buttons-support-markdown

cc @bendansby

### CODE REVIEW

File: src/Nri/Ui/Button/V2.elm [hunk 1]
```diff
@@ -595,17 +597,29 @@ buttonClass size width colorPalette =
...
-                , Html.text label
-                ]
+            [ span [ styles.class [ LabelIconAndText ] ]
+                (decorativeIcon iconType
+                    :: renderMarkdown label
+                )
+            ]
+
+
+renderMarkdown : String -> List (Html msg)
```
Comment (stoeffel) [inline src/Nri/Ui/Button/V2.elm]: 📝 this is used in https://github.com/NoRedInk/elm-formatted-text/blob/master/src/FormattedText/Markdown.elm#L62 as well

**Comment by jwoudenberg** [comment]
Wondering if we should allow buttons (or a button) to take any `Html` content instead.

**Comment by stoeffel** [comment]
@jwoudenberg I think I like this better, because it ensures that markdown is always rendered correctly in buttons.

File: src/Nri/Ui/Button/V2.elm (hunk 1 above)
Comment (brookeangel) [reply src/Nri/Ui/Button/V2.elm]: 👍

**Pull request merged**